modules/ROOT/pages/how-a-bdm-is-deployed.adoc

```diff
@@ -172,8 +174,10 @@ Using the Bonita Portal to deploy a BDM follows this flow:
 * you must be logged in as the tenant administrator
 * go to the BPM services page, and pause the services
 * go to the Business Data Model page, select the BDM zip file to deploy
-* if a BDM already exists on the current tenant, the portal implicitly calls uninstallBusinessDataModel() on the Engine API
-* then the portal calls installBusinessDataModel() on the Engine API, which generates the Java model, loads it in the tenant
+* the portal will call the method updateBusinessDataModel() to install the new BDM.
+This method implicitly calls the two following methods in a single database transaction:
```

Comment:
```suggestion
This method implicitly perform the following steps in a single database transaction:
```

Reply:
I am not sure about the "single database transaction"
why database?

Reply:
~perform~ => perform**s**

Reply:
```suggestion
This method implicitly performs the following steps in a single database transaction:
```